.gitignore

```diff
@@ -31,6 +31,7 @@ Generated.xcconfig
 
 coverage/
 .test_coverage.dart
+coverage_badge.svg
```

Comment:
How is this relevant to us ?

Reply:
If you run test_coverage locally, it gets generated. 🤷‍♂️

Reply:
Umm....ok